### ISSUE TITLE
No XML attribute generated for attribute

### ISSUE THREAD

**Issue by JoopBL** [swagger-api/swagger-core#2089]
I have the following XML:
<xs:element name="CancelRequest" nillable="true" type="CancelRequest" /> <xs:complexType name="CancelRequest"> <xs:sequence> <xs:element minOccurs="1" maxOccurs="1" name="OrderID" type="xs:string" /> </xs:sequence> <xs:attribute name="version" type="xs:string" /> </xs:complexType>
The corresponding Java class has ben generated using xjc:
```java
@XmlAccessorType(XmlAccessType.FIELD)
@XmlType(name = "CancelRequest", propOrder = {
"orderID"
})
@XmlRootElement(name = "CancelRequest")
public class CancelRequest
implements Serializable, ToString
{

private final static long serialVersionUID = -1L;
@XmlElement(name = "OrderID", required = true)
protected String orderID;
@XmlAttribute(name = "version")
protected String version;
```
rest of class are getters and setters

The code generated by Swagger is
"CancelRequest": { "type": "object", "required": [ "orderID" ], "properties": { "orderID": { "type": "string", "xml": { "name": "OrderID" } }, "version": { "type": "string" } }, "xml": { "name": "CancelRequest" } },

There are no XML Object values generated for version. I would have expected here:
"xml": { "attribute":true }
Since version is an attribute.

**Comment by webron** [comment]
I'm afraid there's no way to represent that XML in Swagger (the spec doesn't support it). There's nothing we can do to fix that.

**Comment by JoopBL** [comment]
Is there a way to prevent taking over an attribute?

**Comment by webron** [comment]
I don't understand the question.

**Comment by JoopBL** [comment]
The now generated XML object contains an element (in this case version), which is surely not valid. So I prefer to have no element instead of an wrong element.
So the question is: is it possible to not generate an element for an attribute?

**Comment by JoopBL** [comment]
This issue is solved in the 1.5.12 version
